hmri_check_opt.m

```diff
@@ -0,0 +1,56 @@
+function [opts,ind_repl] = hmri_check_opt(opts_def,opts,loc_opt)
+
+% FORMAT opts = crc_check_flag(opts_def,opts)
+%
+% Function to automatically check the content of a "opts" structure, using
+% a "default opts structure", adding the missing fields and putting in the 
+% default value if none was provided.
+%
+% INPUT:
+% opts_def  : default or reference structure
+% opts      : input option structure that need to be filled for missing
+%             fields with default values
+%
```

Comment:
`loc_opt` input not documented.

Reply:
It is described actually but, admittedly not at the right place, see around lines 22-23.